src/docs/markdown/caddyfile/matchers.md

```diff
@@ -239,7 +306,40 @@ not {
 
 The results of the enclosed matchers will be negated.
 
+#### Examples:
+
+Match requests with paths that do NOT begin with `/css/` OR `/js/`.
+
+```caddy-d
+not path /css/* /js/*
+```
+
+Match requests WITH NEITHER:
+- an `/api/` path prefix, NOR
+- the `POST` request method
+
+i.e. can have none of these to match
+
+```caddy-d
+not path /api/*
+not method POST
+```
+
+Match requests WITHOUT BOTH:
+- an `/api/` path prefix, AND
+- the `POST` request method
+
+i.e. can have zero or one of these to match
```

Comment:
```suggestion
i.e. can match neither or either, but not both:
```

Reply:
I think `i.e. can have neither or either of these to match:` would work.

Reply:
Or what about "must not match both"

Reply:
I think it's important to specify that it can match if zero or one of them match. Otherwise, it's ambiguous what happens in that case if we don't say so.

Reply:
Okay, "can have neither or either of these to match" sounds good to me then.